content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Isito mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+- One or more Kubernetes clusters with versions: {{< supported_kubernetes_versions >}}.
+
+- Authority to [deploy the Istio control plane](/docs/setup/install/istioctl/).
+
+- Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, or alternatively, every pod in the mesh via L3 networking.
+
+- Services in the cluster must be accessible through the ingress gateway.
+
+## Installing the Istio control plane
+
+1. Generate intermediate CA certificates for each cluster's CA from your
+    organization's root CA. The shared root CA enables mutual TLS communication
+    across different clusters or virtual machines. For illustration purposes,
+    the following instructions use the certificates from the Istio samples
+    directory for the cluster.
+
+1. Run the following commands on each cluster in the mesh where a virtual
+     machine connection should be established. See
+     [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+     for more details on configuring an external CA.
+
+{{< text bash >}}
+$ cd samples/certs
+$ make vm
+$ kubectl create namespace istio-system
+$ kubectl create secret generic cacerts -n istio-system \
+    --from-file=vm/ca-cert.pem \
+    --from-file=vm/ca-key.pem \
+    --from-file=vm/root-cert.pem \
+    --from-file=vm/cert-chain.pem
+$ cd ../..
+{{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > $HOME/cluster.env
+    {{< /text >}}
+
+1. If the Kubernetes cluster makes workload requests to the virtual machine,
+   the ports used in the request must be specified in `cluster.env`. An
+   example of exposing the `3306` and `8080` ports on the virtual machine:
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> $HOME/cluster.env
+    {{< /text >}}
+
+1. Install Istio with mesh expansion features:
+
+    {{< text bash >}}
+    $ istioctl install -f install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml
```

Comment:
```
$ ls install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml
ls: cannot access 'install/kubernetes/operator/examples/vm/values-istio-meshexpansion.yaml': No such file or directory
$ find | rg meshexp
./manifests/examples/vm/values-istio-meshexpansion.yaml
./manifests/examples/vm/values-istio-meshexpansion-gateways.yaml
```

Reply:
The IOP I am running with is as follows:
```
sdake@beast-01:~/istio-1.6.0-beta.0/samples/certs$ cat *yaml
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  values:
    global:
      meshExpansion:
        enabled: true
      controlPlaneSecurityEnabled: true
```

Reply:
resolved in prior comment.